src/components/molecules/AnnouncementMessage/AnnouncementMessage.tsx

```diff
@@ -37,7 +36,9 @@ export const AnnouncementMessage: React.FC<AnnouncementMessageProps> = ({
         centered: !isExpanded,
       })}
     >
-      <div className="announcement-message">{getLinkFromText(message)}</div>
+      <div className="announcement-message">
+        <ReactMarkdown>{message}</ReactMarkdown>
```

Comment:
I expect we'll probably want to configure some different props on this to ensure it's more secure/limited to only the things we intend to allow:

- https://github.com/remarkjs/react-markdown#api

We might also want to use `rehype-sanitize` as mentioned in:

- https://github.com/remarkjs/react-markdown#security
  - https://github.com/rehypejs/rehype-sanitize

Reply:
Added rehype-sanitize. Happy to configure additional props, but could use some guidance on which would be necessary here

Reply:
I've now added in props the add URLs directly (to match the prior functionality) and set the target for links to '_blank'

Reply:
Sounds good :)

---

> but could use some guidance on which would be necessary here

I would need to look at the code + options + think about this at a deeper level before i'd be able to give more specific guidance here.

Reply:
may be https://github.com/remarkjs/react-markdown#security would add some confidence:

Use of react-markdown is secure by default. Overwriting transformLinkUri or transformImageUri to something insecure will open you up to XSS vectors. Furthermore, the remarkPlugins and rehypePlugins you use and components you write may be insecure.

and since we are talking about client side, and we verified that known concerns (URLs) are now handled nicely: any specific other concern to check?

Reply:
> and since we are talking about client side

All XSS/etc is client-side by it's nature, that in itself doesn't remove risk.

> and we verified that known concerns (URLs) are now handled nicely: any specific other concern to check?

Not off the top of my head with regards to actual security/etc in this specific part here